invenio_app_rdm/records_ui/utils.py

```diff
@@ -55,3 +58,33 @@ def set_default_value(record_dict, value, path, default_prefix="metadata"):
         path = "{}.{}".format(default_prefix, path)
 
     dict_set(record_dict, path, value)
+
+
+def get_external_resources(record):
+    """Generate external resources from the record."""
+    result = []
+    resources_cfg = current_app.config["APP_RDM_RECORD_LANDING_PAGE_EXTERNAL_LINKS"]
+    for cfg in resources_cfg:
+        try:
+            res = cfg["render"](record)
+            if res:
+                result.append(res)
+        except Exception:
+            pass
+    return list(chain.from_iterable(result))
```

Comment:
Because we could potentially have 3 urls to "render"
2 of them could be github
And 1 openaire
The weirdness (no workaround) is that the renderer say for github will take the 3 urls
And then will output 2 formated dicts
The renderer for openaire will also take the 3 urls
And will only output 1 formated dict
Hence in the end we get something like [[gh1, gh2], [openaire1]] which we need to unnest